solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1158,6 +1159,31 @@ impl pallet_configuration::Config for Runtime {
     type WeightInfo = ();
 }
 
+impl pallet_migrations::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type MigrationsList = ();
```

Comment:
Even though we don't have any migrations yet, we probably want this to be
```suggestion
    type MigrationsList = (tanssi_runtime_common::migrations::StartlightMigrations<Runtime>,);
```

If possible that type should be defined in tanssi_runtime_common crate along DanceboxMigrations and FlashboxMigrations, but if that causes some dependency errors, just copy it to some file in this crate